### PR TITLE
updated github actions competency to include security considerations

### PR DESCRIPTION
**Pull request by wszaranski**

### CODE REVIEW

**Review by codeallthethingz — APPROVED** [review]
Nice addition. 👍

File: competencies/github-actions.md [hunk 1]
```diff
@@ -7,12 +7,15 @@ GitHub Actions are an API for cause and effect on GitHub: orchestrate any workfl
...
 * You can list three events that can trigger a workflow to run and explain the use cases of each.
 * You can list three actions from the GitHub Marketplace and explain the use cases of each.
-* You can expain the differences between workflows, events, jobs, steps, actions, and runners.
+* You can explain the differences between workflows, events, jobs, steps, actions, and runners.
 * You know how to securely store secrets within GitHub, and are aware of the security risks in not doing so.
 * You know how to cache dependencies using `actions/cache@v2` in order to speed up workflows.
 * You know how to add conditional steps, such as `if: github.event.pull_request.draft == false`, to a job.
 * You know how to use `upload-artifact` and `download-artifact` actions in order to persist and share data across workflows.
 * You know how to create workflow templates and use templates created by others.
+* You know how to securely add actions from marketplace
```
Comment (adarj) [inline competencies/github-actions.md:16]: "You know how to securely add actions from the GitHub Marketplace" reads a bit better here.

File: competencies/github-actions.md [hunk 2]
```diff
@@ -7,12 +7,15 @@ GitHub Actions are an API for cause and effect on GitHub: orchestrate any workfl
...
+* You review code of action before you add it to your repository
```
Comment (adarj) [inline competencies/github-actions.md:17]: "You are aware that it's a good practice to review the action's code before you add it to your repository" reads a bit better here.

File: competencies/github-actions.md (hunk 1 above)
Comment (wszaranski) [reply competencies/github-actions.md]: Thanks. That's true, it's more clear now. To be more specific I've added link and info that this applies also to any actions from repositories that you don't own or control.
* You know how to securely add actions from the [GitHub Marketplace](https://github.com/marketplace) or any other git repository that you don't own/control

File: competencies/github-actions.md (hunk 2 above)
Comment (wszaranski) [reply competencies/github-actions.md:17]: Thanks for your feedback. I think that "good practice" sounds like something that you CAN omit while you are developing for example "proof of concept" and adding action from unknown source MAY compromise all of your repositories. If you are sure that your wording is really better I will go with it. Are you sure we should define it just as a "good practice"?

Comment (adarj) [reply competencies/github-actions.md:17]: Yeah I agree with that actually, so "You review an action's code before you add it to your repository" would work better than my earlier suggestion.

**Pull request merged**